x-pack/test/cases_api_integration/common/lib/api/index.ts

```diff
@@ -846,3 +853,128 @@ export const replaceCustomField = async ({
 
   return theCustomField;
 };
+
+export const addObservable = async ({
+  supertest,
+  params,
+  expectedHttpCode = 200,
+  auth = { user: superUser, space: null },
+  headers = {},
+  caseId,
+}: {
+  supertest: SuperTest.Agent;
+  params: AddObservableRequest;
+  expectedHttpCode?: number;
+  auth?: { user: User; space: string | null } | null;
+  headers?: Record<string, string | string[]>;
+  caseId: string;
+}): Promise<Case> => {
+  const apiCall = supertest.post(
+    `${getSpaceUrlPrefix(auth?.space)}${INTERNAL_CASE_OBSERVABLES_URL.replace('{case_id}', caseId)}`
```

Comment:
nit: You can use the helpers you created to construct the URLs. Example: `getCaseCreateObservableUrl`. Same for the other functions below.